doc/src/Developer_cxx_vs_c_style.rst

```diff
@@ -190,96 +185,105 @@ are virtual functions that are initialized to 0 in the class declaration
-any destructors in classes containing virtual functions should be
-declared virtual, too, so they are processed in the expected order
-before types are removed from dynamic dispatch.
+derived class and thus it is good practice to either avoid calling
+them or to provide an explicit scope such as ``Base::poly()``.
+Furthermore, any destructors in classes containing virtual functions
+should be declared virtual, so they will be processed in the expected
+order before types are removed from dynamic dispatch.
 
 .. admonition:: Important Notes
 
-   In order to be able to detect incompatibilities and to avoid unexpected
-   behavior already at compile time, it is crucial that all member functions
-   that are intended to replace a virtual or pure function use the ``override``
-   property keyword.  For the same reason it should be avoided to use overloads
-   or default arguments for virtual functions as they lead to confusion over
-   which function is supposed to override which and which arguments need to be
-   declared.
+   In order to be able to detect incompatibilities at compile time and
+   to avoid unexpected behavior, it is crucial that all member
+   functions that are intended to replace a virtual or pure function
+   use the ``override`` property keyword.  For the same reason, the
+   use of overloads or default arguments for virtual functions should
+   be avoided as they lead to confusion over which function is
+   supposed to override which and which arguments need to be declared.
 
 Style Factories
 ===============
 
-In order to create class instances of the different styles, LAMMPS often
-uses a programming pattern called `Factory`.  Those are functions that create
-an instance of a specific derived class, say ``PairLJCut`` and return a pointer
-to the type of the common base class of that style, ``Pair`` in this case.
-To associate the factory function with the style keyword, an ``std::map``
-class is used in which function pointers are indexed by their keyword
-(for example "lj/cut" for ``PairLJCut`` and "morse" ``PairMorse``).
-A couple of typedefs help to keep the code readable and a template function
-is used to implement the actual factory functions for the individual classes.
+In order to create class instances for different styles, LAMMPS often
+uses a programming pattern called `Factory`.  Those are functions that
+create an instance of a specific derived class, say ``PairLJCut`` and
+return a pointer to the type of the common base class of that style,
+``Pair`` in this case.  To associate the factory function with the
+style keyword, an ``std::map`` class is used with function pointers
+indexed by their keyword (for example "lj/cut" for ``PairLJCut`` and
+"morse" for ``PairMorse``).  A couple of typedefs help keep the code
+readable and a template function is used to implement the actual
+factory functions for the individual classes.
+
+NOTE to Axel: point to a couple of files that have these factory
```

Comment:
See this NOTE as a suggestion for something to add

Reply:
Yup. I had planned to put something here, but then deferred, because I was doing a little refactor and simplification in another pull request.